packages/sdk/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "root": true,
+  "extends": ["@across-toolkit/eslint-config/library.js"],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": true
+  },
+  "env": {
+    "es2020": true
+  }
+}
```

Comment:
Do you know if there is some rule to enforce the `.js` extension when importing? 🤔

Reply:
I believe the typescript parser for eslint will complain if you don't do, while "module" is set to "NodeNext" in the tsconfig

![Screenshot 2024-11-05 at 10 43 49](https://github.com/user-attachments/assets/e41d6033-7e6f-4681-891f-7e70ce656985)